docs/RELEASE_POLICY.md

```diff
@@ -90,6 +90,7 @@ v3.1 will be patched since were the last two Firecracker releases and less than
 
 | Release | Release Date | Latest Patch | Min. end of support | Official end of Support         |
 | ------: | -----------: | -----------: | ------------------: | :------------------------------ |
+|    v1.9 |   2024-09-01 |       v1.9.0 |          2025-03-01 | Supported                       |
```

Comment:
```suggestion
|    v1.9 |   2024-09-02 |       v1.9.0 |          2025-03-01 | Supported                       |
```
September 1st is a sunday :D

Reply:
I could have sworn I checked exactly for this. More coffee it is...